golang/connection.go

```diff
@@ -552,7 +553,11 @@ func (c *Connection) SendSystemError(id uint32, span *Span, err error) error {
 
 // connectionError handles a connection level error
 func (c *Connection) connectionError(err error) error {
-	c.log.Warnf("Connection error: %v", err)
+	if err == io.EOF {
+		c.log.Debugf("Connection got EOF")
```

Comment:
I wanted to make it explicit, because when connection.Close is called, or channel starts graceful close, it logs things like "Channel Close". This way, it's explicit and kept separate from that Close